### ISSUE TITLE
There Should be a Way to Reopen the Round End Screen after it Closes

### ISSUE THREAD

**Issue by Geekyhobo** [space-wizards/space-station-14#18031]
## Description

See title

**Comment by JoeHammad1844** [comment]
true

**Comment by UbaserB** [comment]
I totally agree on this, sometimes i subconsciously close the menu because i want to see whats going on, and then i realise what i did and i get depressed

**Comment by theo-pirkl** [comment]
Hey, I realise this is old, but you now can if you change the keybindings. There is a "toggle round end summary window". I agree that adding a button would help making this feature

**Comment by osjarw** [comment]
Fixed by #25884

**Comment by PJB3005** [comment]
> Fixed by #25884

Having a hotkey is nice, but I would really like a better way in the UI too.

**Comment by magelfik** [comment]
*woosh* there you go ! Hopefully it is up to SS14's standards. Feel free to comment !

**Comment by SlamBamActionman** [comment]
Exists now, F9
